BUILD

```diff
@@ -0,0 +1,614 @@
+# This Package contains the necessary targets to incorporate the googletest
+# libraries into your Bazel built projects.
+#
+# The libraries are defined here rather than in their component directories
+# because googlemock has dependencies on googletest and by defining the
+# build targets here, the libraries are independent of their location in
+# a more straightforward way.
+
+licenses(["notice"])
+
+cc_library(
+    name = "googletest",
+    srcs = glob([
+    	"googletest/src/gtest.cc",
+    	"googletest/src/gtest-death-test.cc",
+    	"googletest/src/gtest-filepath.cc",
+    	"googletest/src/gtest-internal-inl.h",
+    	"googletest/src/gtest_main.cc",
```

Comment:
Can you split gtest_main.cc into its own separate target?

We have a library that needs to include gtest to get the FRIEND_TEST macro, but we don't want a main in that situation.